style.css

```diff
@@ -0,0 +1,34 @@
+html {
+  background-color: #FEF9E7;
+}
+
+@media all and (min-width: 640px) {
```

Comment:
Nice start here with the media query! Try adding 'links' for an About page. We'll be toggling this single page app with multi-page functionality throughout the course.